examples/common_vmseries/README.md

```diff
@@ -519,18 +512,11 @@ map(object({
 
 
 
-### Optional Inputs
-
-
-#### tags
 
-Map of tags to assign to the created resources.
 
-Type: map(string)
 
-Default value: `map[]`
 
-<sup>[back to list](#modules-optional-inputs)</sup>
+### Optional Inputs
```

Comment:
why do we have `###` 3 hashes in some comments in `####` 4 hashes in some like in "name_prefix"? is there a hierarchy?

Reply:
Yes, it is a hierarchy, that's how you mark headers in MD files. Take a look at the README file in Preview mode.